debug_toolbar/toolbar.py

```diff
@@ -200,7 +207,9 @@ def from_store_get_response(request):
 
 
 class StoredDebugToolbar(DebugToolbar):
-    def __init__(self, request, get_response, request_id=None):
+    def __init__(
+        self, request: HttpRequest, get_response: "GetResponse", request_id=None
```

Comment:
```suggestion
        self, request: HttpRequest, get_response: GetResponse, request_id=None
```

I think this is also missing the type definition for `request_id`